yazi-dds/src/client.rs

```diff
@@ -62,6 +63,56 @@ impl Client {
 		});
 	}
 
+	/// Connect to an existing server and listen in on the messages that are being
+	/// sent by other yazi instances.
+	/// If no server is running, fail right away.
+	/// If a server is closed, attempt to reconnect forever.
+	pub async fn echo_events_to_stdout(kinds: HashSet<String>) -> Result<()> {
+		let mut lines = Self::connect_listener(&kinds).await?;
+		let mut msg_counter = 0;
+
+		loop {
+			println!("Waiting for messages...");
+			match lines.next_line().await {
+				Ok(Some(s)) => {
+					println!("Received: message {} '{}'", msg_counter, s);
```

Comment:
These logs should be removed

Reply:
The logs have been removed in 3e9a0d95bd2387dbccdb0faa7093fe8ad36047dd